gradle.properties

```diff
@@ -1 +1 @@
-version=0.0.23-RC1
+version=0.1.0-beta1
```

Comment:
Can you revert this? I think it is better if we raise the version in a separate PR.